src/home_robot/home_robot/perception/detection/lseg/modules/lsegmentation_module.py

```diff
@@ -11,14 +11,15 @@
 
 # add mixed precision
 import torch.cuda.amp as amp
-from data import get_available_datasets, get_dataset
 from encoding.nn import SegmentationLosses
 from encoding.utils import (
     SegmentationMetric,
     batch_intersection_union,
     batch_pix_accuracy,
 )
 
+from data import get_available_datasets, get_dataset
```

Comment:
This will cause linter issues on CircleCI as well. Just change it back to what it was.

Reply:
Fixed